engine/src/gui/text_area.cpp

```diff
@@ -181,21 +180,18 @@ void TextArea::Refresh(void) {
     }
 }
 
-void TextArea::RenderText(void) {
```

Comment:
This whole code feels like it was taken from some C program or written by some C programmer. `RenderText(void)` and init all variables at the start of the function... weird.